apps/signal/.env.example

```
@@ -1,9 +1,10 @@
 # Port for the signaling server
 PORT=3001
 
-# Supabase (for voice_states sync + auth verification)
-SUPABASE_URL=http://your-coolify-domain:8000
+# Supabase Cloud — get from app.supabase.com → your project → Settings → API
+SUPABASE_URL=https://<project-ref>.supabase.co
 SUPABASE_SERVICE_ROLE_KEY=your-service-role-key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SUPABASE_SERVICE_ROLE_KEY key should go before the SUPABASE_URL key

(UnorderedKey)